src/Microsoft.Health.Fhir.Api/Features/Routing/RouteNames.cs

```diff
@@ -7,68 +7,74 @@ namespace Microsoft.Health.Fhir.Api.Features.Routing
 {
     internal static class RouteNames
     {
-        internal const string Metadata = "Metadata";
+        internal const string Metadata = nameof(Metadata);
```

Comment:
Trying to reason if this is a change for the better, it makes sense that it can be the same name as the property, though the compile-time output will be the same. The advantage of `nameof` is that is responds to refactoring within the IDE tools, but in this case doesn't seem desired